src/com/leetcode/problems/StockProfit.java

```diff
@@ -0,0 +1,22 @@
+package com.leetcode.problems;
+
+// https://leetcode.com/problems/best-time-to-buy-and-sell-stock/
+
+public class StockProfit {
+  public static void main(String[] args) {
+    System.out.println((maxProfit(new int[] {7, 6, 4, 3, 1})));
+  }
+
+  public static int maxProfit(int[] prices) {
+    int profitToday = 0, mediumProfit = 0;
+    int minPrice = Integer.MAX_VALUE;
```

Comment:
Does it make sense to you that the name is "minPrice" but the value is  "MAX_VALUE" of int?